src/Common/Helper.tsx

```diff
@@ -367,3 +368,75 @@ export function useSearchString(): UseSearchString {
 
     return { queryParams, searchParams }
 }
+
+export const getCDMaterials = (
```

Comment:
this should be in some service as we do API calls from service only

Reply:
done